web/styles/portico/navbar.css

```diff
@@ -0,0 +1,570 @@
+    background-position: center;
+    cursor: pointer;
+    opacity: 0.5;
+}
+
+.top-menu__tab-label-unselect:hover {
+    opacity: 0.8;
+}
+
+.top-menu__tab-label-unselect:active {
+    opacity: 1;
+}
+
+.top-menu__tab-input-unselect:not(:checked) ~ .top-menu__tab-label-unselect {
+    display: block;
+}
+
+.top-menu__submenu {
+    position: absolute;
+    top: 60px;
+    display: flex;
+    gap: 16px;
+    opacity: 0;
+    visibility: hidden;
+    transition: all 0.2s;
+}
+
+.top-menu__tab .top-menu__tab-user--label {
+    max-width: 140px;
+    padding-right: 28px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.top-menu__tab-user--img {
+    width: 24px;
+    height: 24px;
+    display: inline;
+    vertical-align: middle;
+    border-radius: 3px;
+    margin-right: 4px;
+}
+
+.top-menu-mobile__user--img {
+    width: 28px;
+    height: 28px;
+    display: inline;
+    vertical-align: text-top;
+    border-radius: 3px;
+    margin-right: 4px;
+}
+@media (max-width: 1400px) {
+    .top-menu__submenu._right-aligned {
+        justify-content: flex-end;
+        right: 44px;
+    }
+}
+@media (max-width: 1170px) {
+    .top-menu__tab .top-menu__tab-user--label {
+        max-width: 84px;
+    }
+}
+@media (max-width: 1024px) {
+    .top-menu__tab .top-menu__tab-user--label {
+        max-width: 40px;
+    }
+
+    .top-menu__tab .top-menu__tab-user--label span {
+        display: none;
+    }
+}
+
+.top-menu__tab-input:checked ~ .top-menu__submenu {
+    opacity: 1;
+    visibility: visible;
+}
+
+.top-menu__submenu-column {
+    display: flex;
+    flex-direction: column;
+    padding-top: 16px;
+    flex-shrink: 0;
+    max-width: 420px;
+}
+
+.top-menu__submenu-section {
+    padding-left: 16px;
+    font-size: 17px;
+    opacity: 0.6;
+    margin-bottom: 4px;
+    letter-spacing: 2px;
+}
+
+.top-menu__submenu-column a {
+    padding: 6px 16px;
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 125%;
+    color: hsla(0, 0%, 100%, 0.9);
+    border-radius: 2px;
+    cursor: pointer;
+}
+
+.top-menu__submenu-column a:hover {
+    background: hsla(0, 0%, 100%, 0.1);
+}
+
+.top-menu__submenu-column a:active {
+    background: hsla(0, 0%, 100%, 0.2);
+}
+
+.top-menu-mobile {
+    display: none;
+    background: hsla(240, 100%, 10%, 0.4);
+    backdrop-filter: blur(10px);
+    z-index: 20;
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 60px;
+
+    overflow: auto;
+    font-family: var(--font-ss3);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 28px;
+    letter-spacing: 0.02em;
+    font-feature-settings: "pnum" on, "lnum" on;
+    color: hsla(0, 0%, 100%, 0.8);
+
+    transition: all 0.2s;
+    overscroll-behavior: contain;
+}
+
+.top-menu-mobile[open] {
+    bottom: 0;
+    /* mobile viewport bug fix */
```

Comment:
Comments like this would be a lot better with more explanation of what the bug is. We almost never want to have "bug fix" be in a comment. Something like "Prevent the mobile viewport from ..." would be a lot more helpful.

Reply:
@terpimost FYI

Reply:
Using `height: 100%` works, well here, I am not sure what @terpimost was trying to fix.

Reply:
I understand what you are trying to tell. if something works, lets use it.